Dockerfile

```diff
@@ -4,6 +4,7 @@ ARG USER_UID=1000
 ARG USER_GID=$USER_UID
 
 FROM ubuntu:20.04 AS gadgetron_baseimage
+LABEL org.opencontainers.image.source=https://github.com/gadgetron/gadgetron
```

Comment:
Will this label be applied to the actual images we publish (the runtime and dev images)?

Reply:
When I run this locally and inspect the images, each of the four actual images still has the label, if the label is added at the top (which should be the desired bahaviour I think, since all four images are from this repo).  I'm not sure how GitHub uses/respects the labels though, so partly I want to see what happens in the GitHub UI with this PR, then we can add more labels to other layers that might help improve package version searching etc.